ci/set_kubectl_config-azure.sh

```diff
@@ -0,0 +1,9 @@
+KUBE_CLUSTER=$1
```

Comment:
I think we could pass AZURE_SP_TENANT, AZURE_CLIENT_ID, AZURE_SECRET, RESOURCE_GROUP instead of the KUBE parameters. unless we are okay with global env (in the previous version I passed them as parameters to make it somewhat more understandable but in practice I suppose it doesn't make huge diffrerence).